its/org.sonarlint.eclipse.its.connected.sc/build.properties

```diff
@@ -0,0 +1,4 @@
+source.. = src/
+output.. = target/classes/
+bin.includes = META-INF/,\
+               .
```

Comment:
Strange formatting

Reply:
Yes for Eclipse this is normal :smile: and required :face_in_clouds: